people/qiqi233345/page.en.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Qi
+---
+Qi doesn't often talk on the Internet, but she is very enthusiastic in real life. When being with her, the funny she show always bring people happiness.
+She looks mysterious, and doesn't introduce herself very much because she thinks "I am me, you just know "me", you don't need to know "me with a certain label". If someone can't understand me, that's scolding me, don’t like me. I don't want he to hate people who have the same label or experience as me because of me.”
+Qi is a gentle, brave, resolute person who dares to love and hate. She will bravely stand up to speak for transgender people, and patiently comfort those who are sad. She has many unique and sharp insights, and she is also very powerful that she do not allow others to hurt your friends and lovers. She also can't be hurt by others.
+But it is a pity that she was also plagued by depression and anxiety. On March 14, 2023, she attempted suicide by jumping off a building due to a depression attack, and disabled her legs. This made her life very inconvenient and aggravated her depression.
+On July 1, 2023, she jumped off the building again after taking a large amount of antidepressant drugs, but the rescue failed and died.
```

Comment:
On July 1, 2023, she jumped off a building again after taking a large amount of antidepressant drugs.
This time, she couldn't be saved.

Reply:
her injuries were beyond saving?